ci-jobs/templates/ios-e2e-template.yml

```diff
@@ -70,6 +73,7 @@ jobs:
           --port=$APPIUM_TEST_SERVER_PORT \
           --address=$APPIUM_TEST_SERVER_HOST \
           --relaxed-security \
+          --keep-alive-timeout 1200 \
```

Comment:
The default is 600. It turned out that `POST /session` takes longer[ with Xcode 14.2 + iOS 16. ](https://dev.azure.com/AppiumCI/7a2a2919-d2b4-4720-a6c3-3a639f9bb2e0/_apis/build/builds/24486/logs/133). E.g. boot time gets 1.2x - 1.5x slower than Xcode 13 + iOS 15
> UDID FF980442-1833-4484-8258-3B44A8458870 booted in 333.876s

Now it takes more than 10 minutes to create a new session and test failed due to disconnection from the server (socket hang up)
```
2023-01-05T05:55:16.5477380Z [38;5;0m[HTTP][0m [37m-->[39m [37mPOST[39m [37m/session[39m
2023-01-05T06:05:16.1550910Z [38;5;0m[HTTP][0m <-- POST /session - - ms - -
```
```
2023-01-05T06:05:16.1856360Z [90m2023-01-05T06:05:16.178Z[39m [31mERROR[39m [97mwebdriver[39m: RequestError: socket hang up
2023-01-05T06:05:16.1958370Z     at ClientRequest.<anonymous> (file:///Users/runner/work/1/s/node_modules/webdriver/node_modules/got/dist/source/core/index.js:789:107)
```
So I extended the timeout.